docs(gen): docs content review - week 33/2023 (#2043)
scaleway/docs-content: tutorials/encrypt-s3-data-rclone/index.mdx

    \ "standard"
  2 / Very simple filename obfuscation.
    \ "obfuscate"

>   3 / Do not encrypt the file names.  Adds a ".bin" extension only.

Reviewer: Should this be "Add" ?

Author: This a copy of the output of the tool. I re-edit the message back to what it was

>   3 / Do not encrypt the file names.  Adds a ".bin" extension only.

Author: ```suggestion
  3 / Don't encrypt the file names.  Adds a ".bin" extension only.
```
